core/input/input_map.cpp

```diff
@@ -48,7 +48,7 @@ void InputMap::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("action_has_event", "action", "event"), &InputMap::action_has_event);
 	ClassDB::bind_method(D_METHOD("action_erase_event", "action", "event"), &InputMap::action_erase_event);
 	ClassDB::bind_method(D_METHOD("action_erase_events", "action"), &InputMap::action_erase_events);
-	ClassDB::bind_method(D_METHOD("get_action_list", "action"), &InputMap::_get_action_list);
+	ClassDB::bind_method(D_METHOD("get_action_events", "action"), &InputMap::_get_action_events);
```

Comment:
I think it might be better to name it `action_get_events` to match the convention used in other methods of this class (all methods which relate to a specific action start with `action_`, like `action_has_event`).